ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1443,7 +1423,7 @@ void Copter::mavlink_delay_cb()
     uint32_t tnow = millis();
     if (tnow - last_1hz > 1000) {
         last_1hz = tnow;
-        gcs_send_heartbeat();
```

Comment:
These are completely different animals.  One queues, one sends.

Also, this will lead to uart overruns.

Reply:
Yes, this line gives
```
ArduCopter/GCS_Mavlink.cpp:1426:15: error: 'class GCS_Copter' has no member named 'send_heartbeat'
         gcs().send_heartbeat();
```
😅

Reply:
that is existing code in the callback, and the callback only happens when we are not actively sending

Reply:
i don't think this can lead to uart overruns, as it is only called when link is idle (in delay cb during startup)